build/build.go

```diff
@@ -79,6 +79,11 @@ func importWithSrcDir(path string, srcDir string, mode build.ImportMode, install
 		return nil, err
 	}
 
+	// TODO: Resolve issue #415 and remove this temporary workaround.
+	if strings.HasSuffix(pkg.ImportPath, "/vendor/github.com/gopherjs/gopherjs/js") {
```

Comment:
I know `pkg.ImportPath` will always be an absolute import path, but I'm not sure about `path`. It might be relative. E.g., consider:

```
cd $GOPATH/src/github.com/gopherjs/gopherjs
gopherjs install ./js
```

Is there some issue with this method, or do you dislike it because of a string comparison?

I _think_ this is the canonical way to detect vendored packages, but I could be wrong.